src/onegov/org/models/page.py

```diff
@@ -42,6 +42,8 @@ class Topic(Page, TraitInfo, SearchableContent, AccessExtension,
     # Show the lead on topics page
     lead_when_child = content_property(default=True)
 
+    western_name_order = content_property(default=False)
```

Comment:
Put this in `PersonLinkExtension` not `Topic`, `Topic` inherits from `PersonLinkExtension`, but other models do too, the form will break with those models. E.g. `News` won't work correctly now.

Reply:
Yes, got it!